src/shared_modules/keystore/src/keyStore.cpp

```diff
@@ -40,6 +40,38 @@ void Keystore::put(const std::string& columnFamily, const std::string& key, cons
     }
 }
 
-void Keystore::get(const std::string& columnFamily, const std::string& key, rocksdb::PinnableSlice& value)
+void Keystore::get(const std::string& columnFamily, const std::string& key, std::string& value)
 {
+    std::string encryptedValue;
+
+    try
+    {        
+        // Get from DB
+        Utils::RocksDBWrapper keystoreDB = Utils::RocksDBWrapper(DATABASE_PATH, false);
+
+        if(!keystoreDB.columnExists(columnFamily)) {
+            char msg[100];
+            sprintf(msg, "Column %d does not exists in the database.", columnFamily.c_str());
```

Comment:
I think is nittier `Column %d does not exists at the database` instead of `Column %d does not exists in the database`